flink-table-store-connector/pom.xml

```diff
@@ -151,6 +151,27 @@ under the License.
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-mapreduce-client-core</artifactId>
+            <version>${hadoop.version}</version>
+            <scope>provided</scope>
```

Comment:
Could the scope set to `test`?

Reply:
I also set the scope to `test` at first, but I found that it still throws an exception, so I update it to `provided`  .  I try `test` scope again ,